artifactory/commands/golang/go_test.go

```diff
@@ -21,3 +27,18 @@ func TestBuildPackageVersionRequest(t *testing.T) {
 		})
 	}
 }
+
+func TestGetPackageFilesPath(t *testing.T) {
```

Comment:
This is not a unit test, but an integration test. It requires "go" and also Artifactory. The jfrog-cli-core test command hoeever do not aware of Artifactory, and also do not have requirement to have "go" installed on the test machine (see the Tests section in the README).
This test should probably move jfrog-cli, and be added to go integration tests.

In additopn to the above, I think that the ```getFileSystemPackagePath``` does require a unit test. In order to unit test this function, we don't need "go" or even the file system. It should be "table tested" to see that its logic is good.